src/index.js

```diff
@@ -1,13 +1,46 @@
-import { scores } from './modules/scores.js';
+import UI from './modules/UI.js';
+import Game from './modules/Games.js';
 import './style.css';
 
-function addScore(score) {
-  const list = document.querySelector('.score-list');
-  const li = document.createElement('li');
-  li.innerHTML = `${score.name}: ${score.score}`;
-  list.appendChild(li);
-}
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const id = 'oQNAEST2apGVZIoxn0tw';
+const url = `${baseURL}games/${id}/scores`;
 
-scores.forEach((i) => {
-  addScore(i);
-});
\ No newline at end of file
+const ui = new UI();
+const newGame = new Game();
+
+/* const gameInfo = newGame.addGame({name: 'Gena'}, `${baseURL}games/`);
+let gameId;
+gameInfo.then((result) => {
+  let theStr = result[Object.keys(result)[0]].split(' ');
+  gameId =  theStr[theStr.length-2]
+  console.log('toString: ' + gameId);
+}); */
```

Comment:
- [ ] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1: